bouture.js

```diff
@@ -63,6 +64,17 @@ function completeElement (tag, args) {
               return name.replace(/^once|on/, '').toLowerCase()
             }
 
+            function isAttribute (name) {
```

Comment:
This function is contextual to where it's defined. If `tag` was passed as an argument, the function could be moved to its own file (or at the top of this file)

I feel `isAttribute` is a bit light. Maybe `isValidAttributeName` a better function name